.gitignore

```diff
@@ -1,3 +1,10 @@
+# We have `ansible.cfg` in VCS, but because one may make local changes, we don't
```

Comment:
Or part of a make shell, copy over an ansible.cfg.sample. Only the sample is checked in git and allow users to modify the ansible.cfg at will (or loose it if making another git clone)

Reply:
Yeah, that's a sensible approach (though I wouldn't let `make shell` create a copy, that's a bit IMHO).